src/v2/Apps/Consign/Routes/SubmissionFlow/Utils/phoneNumberUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { Environment, fetchQuery, graphql } from "react-relay"
+import { phoneNumberUtils_PhoneNumber_Query } from "v2/__generated__/phoneNumberUtils_PhoneNumber_Query.graphql"
+
+export const getPhoneNumberInformation = async (
+  phoneNumber: string,
+  relayEnvironment: Environment,
+  regionCode?: string
+) => {
+  const response = await fetchQuery<phoneNumberUtils_PhoneNumber_Query>(
+    relayEnvironment,
+    graphql`
+      query phoneNumberUtils_PhoneNumber_Query(
+        $phoneNumber: String!
+        $regionCode: String
+      ) {
+        phoneNumber(phoneNumber: $phoneNumber, regionCode: $regionCode) {
+          isValid
+          international: display(format: INTERNATIONAL)
+          national: display(format: NATIONAL)
+          originalNumber
+        }
```

Comment:
**suggestion:** Would it make sense to query for errors and display those to the user?